Core/src/main/java/com/paypal/android/core/Environment.kt

```diff
@@ -1,7 +1,16 @@
 package com.paypal.android.core
 
-enum class Environment(val url: String) {
-    LIVE("https://api.paypal.com"),
-    SANDBOX("https://api.sandbox.paypal.com"),
-    STAGING("https://api.msmaster.qa.paypal.com")
+enum class Environment(val url: String, val grqphQlUrl: String) {
+    LIVE(
+        "https://api.paypal.com",
+        "https://www.paypal.com/graphql"
```

Comment:
Random nitpick but do we need `www` on each of these? For consistency can we leave them off?

Reply:
great suggestion!